Library/Homebrew/rubocops/lines_cop.rb

```diff
@@ -79,7 +79,7 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             find_instance_method_call(body_node, "OS", method_name) do |check|
               problem "Don't use #{check.source}; Homebrew/core only supports macOS"
             end
-          end
+          end if OS.mac?
```

Comment:
Please add `next unless OS.mac?` after line 77, before the existing `next unless`.

Reply:
I turns out that `OS.mac?` is not available inside this function, that is, when running Rubocop. This file is a plugin to Rubocop, and is not run inside the usual Brew environment.